docusaurus/src/components/ConnectorRegistry.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { useEffect, useState } from "react";
+
+const oss_registry_url =
+  "https://connectors.airbyte.com/api/v0/catalog/oss_catalog.json";
+const cloud_registry_url =
+  "https://connectors.airbyte.com/api/v0/catalog/cloud_catalog.json";
+const iconBase =
+  "https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-config-oss/init-oss/src/main/resources/icons";
+const iconStyle = { maxWidth: 25 };
+const sourceBase =
+  "https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors";
+const bugsBase =
+  "https://github.com/airbytehq/airbyte/issues?q=is:open+is:issue+label"; // :connectors/source/activecampaign
+
+async function fetchCatalog(url, setter) {
+  const response = await fetch(url);
+  const registry = await response.json();
+  setter(registry);
+  return registry;
+}
+
+/*
+Sorts connectors by release stage and then name
+*/
+function connectorSort(a, b) {
+  if (a.releaseStage !== b.releaseStage) {
+    if (a.releaseStage === "generally_available") return -3;
+    if (b.releaseStage === "generally_available") return 3;
+    if (a.releaseStage === "beta") return -2;
+    if (b.releaseStage === "beta") return 2;
+    if (a.releaseStage === "alpha") return -1;
+    if (b.releaseStage === "alpha") return 1;
+  }
+
+  if (a.name < b.name) return -1;
+  if (a.name > b.name) return 1;
+}
+
+export default function ConnectorRegistry({ type }) {
+  const [ossRegistry, setOssRegistry] = useState([]);
+  const [cloudRegistry, setCloudRegistry] = useState([]);
+
+  useEffect(async () => {
+    await fetchCatalog(oss_registry_url, setOssRegistry);
+    await fetchCatalog(cloud_registry_url, setCloudRegistry);
+  }, []);
```

Comment:
nit: we can drop the async
```suggestion
  useEffect(() => {
    fetchCatalog(oss_registry_url, setOssRegistry);
    fetchCatalog(cloud_registry_url, setCloudRegistry);
  }, []);
```